src/aerospike_clj/client.clj

```diff
@@ -63,27 +64,31 @@
   \"username\"
   :port (default is 3000)
   :client-events an implementation of ClientEvents. Either a single one or a vector
-  thereof. In the case of a vector, the client will chain the instances by order."
+  thereof. In the case of a vector, the client will chain the instances by order.
+  If no event-loops instance is provided, single-threaded one is created automatically."
   ([hosts aero-ns]
    (init-simple-aerospike-client hosts aero-ns {}))
   ([hosts aero-ns conf]
    (let [cluster-name (utils/cluster-name hosts)
-         event-loops (:event-loops conf (create-event-loops conf))
+         event-loops (:event-loops conf)
+         close-event-loops? (nil? event-loops)
```

Comment:
maybe this should default to `true` and can be passed `{:close-event-loops? false}` to explicitly close it?
this couples the event-loops creation to its auto-closing.

Reply:
OTOH, the creator should destroy it as a good thumb rule.

Reply:
Discussed in offline, keeping as is